### ISSUE TITLE
About TSNSE in DM

### ISSUE THREAD

**Issue by VincenDen** [VICO-UoE/DatasetCondensation#27]
Hello,
       In Figure3 T-SNSE of the DM paper， how to extract the features of synthetic images，does it extract features from a model trained on whole dataset or from a model trained on synthetic images?
      Thanks.

**Comment by PatrickZH** [comment]
Hi, we extract the features (last-layer logits) with a ResNet trained on whole original dataset.
